test/data/specific_fields/module/package.json

```diff
@@ -0,0 +1,57 @@
+{
```

Comment:
Is there any reason to add this file instead of using ` test/data/underscore_fields/module/package.json`??

Reply:
There is not.  I was thinking at first that it would modify the file.

Reply:
Please remove this file.